contracts/src/StakeTable.sol

```diff
@@ -90,10 +90,11 @@ contract StakeTable is AbstractStakeTable {
         return keccak256(abi.encode(blsVK.x0, blsVK.x1, blsVK.y0, blsVK.y1));
     }
 
+    /// TODO handle this logic more appropriately when epochs are re-introduced
     /// @dev Fetches the current epoch from the light client contract.
     /// @return current epoch (computed from the current block)
-    function currentEpoch() public view returns (uint64) {
-        return lightClient.currentEpoch();
+    function currentEpoch() public pure returns (uint64) {
```

Comment:
Why do we need to keep the function `currentEpoch()`?

Reply:
at the time i made the change, i wanted to keep the related tests with as little breaking changes as possible - but now that i removed the stateTable tests, i can remove that function

Reply:
i also agree to keep this, a dummy logic is perfect here. Plus we are not using or auditing this contract anyway.